lib/abci/handlers/endBlockHandlerFactory.js

```diff
@@ -68,7 +85,37 @@ function endBlockHandlerFactory(
     const header = blockExecutionContext.getHeader();
     const coreChainLock = latestCoreChainLock.getChainLock();
 
-    let response = {};
+    let newValidatorSet = null;
+    let newValidatorsUpdates = null;
+    // validator set is rotated every ROTATION_BLOCK_INTERVAL blocks
+    if (height % ROTATION_BLOCK_INTERVAL === 0) {
+      const smlStore = simplifiedMasternodeList.getStore();
+      if (smlStore === undefined) {
```

Comment:
I think it doesn't make sense to check it here since we ensure it presents on begin block

Reply:
You're right! I removed the check.